src/ol/reproj/DataTile.js

```diff
@@ -510,7 +510,7 @@ class ReprojDataTile extends DataTile {
     } else {
       this.reprojData_ = dataR;
     }
-    this.reprojSize_ = [Math.round(outWidth), Math.round(outHeight)];
+    this.reprojSize_ = [outWidth, outHeight];
```

Comment:
Do we need to round here again? `outWidth` and `outHeight` should be rounded already.

Reply:
We're not rounding anymore, unless I'm missing something?

Reply:
Sorry @jahow, too much looking at code this Sunday, you're absolutely right.